Web_Development/Javascript/Topics/5.Syntax/2.Operators/3.Comparison_Operators/README.md

```diff
@@ -0,0 +1,75 @@
+# Comparison Operators
+[Back to Operators in Javascript](../README.md#operators-in-javascript)
+
+
+Comparison operators are used to compare values and returns boolean value (true or false).
+
+#### 1. Equal To Operator (==)
+It will return true if both values are equal, otherwise false.
+
+```js
+console.log(5==5); // true
+console.log(4==3); // false
+```
+
+#### 2. Not Equal To Operator (!=)
+It will return true if both values are not equal, otherwise false.
+
+```js
+console.log(5!=5); // false
+console.log(0!='0'); // return true because left value is a number and right value is a string.
```

Comment:
`// return true even if the data types are different but the values are same on the left hand side and right hand side`